### PR TITLE
Add CSC22 page

### PR DESCRIPTION
**Pull request by strieflin**
Signed-off-by: Sven Trieflinger <sven.trieflinger@de.bosch.com>

### CODE REVIEW

**Comment by strieflin** [comment]
@sbckr, @jaredweinfurtner, @chris-zimm: Happy to receive your feedback on this PR.

**Comment by chris-zimm** [comment]
looks great to me

File: docs/community/csc/overview.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Getting There
+
+The Connectory is located in the heart of Stuttgart's city center (see
+[Google Maps][google-maps]). The easiest way to get there is by public
+transport: :material-train: train station _Stadtmitte_ and :material-subway:
+subway station _Rotebühlplatz_ are right around the corner.
+
+[connectory-stuttgart]: https://stuttgartconnectory.com/
+[connectory-stuttgart-img]: https://stuttgartconnectory.com//wp-content/uploads/sites/24/2020/11/Image-9.png
```
Comment (sbckr) [inline docs/community/csc/overview.md:59]: are we allowed to share this image?

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**